app/poros/search.rb

```diff
@@ -0,0 +1,16 @@
+class Search
+  attr_reader :total_results,
+              :query
+  
+  def initialize(results)
+    @total_results = results[:totalHits]
+    @query = results[:foodSearchCriteria][:query].capitalize
+    @all_foods = results[:foods]
+  end
+
+  def foods
+    @all_foods[0..9].map do |food|
```

Comment:
I'm glad you only created 10 Food object. This is efficient. And, this is the right place to do this.